lib/brute/wp_xmlrpc/engine.py

```diff
@@ -86,7 +83,7 @@ def check(user, passwd, target, port, headers, timeout_sec, log_in_file, languag
 
 
 def test(target, port, retries, timeout_sec, headers, socks_proxy, verbose_level, trying, total_req, total, num, language):
-    if socks_proxy is not None:
+    if socks_proxy != None:
```

Comment:
@aman566  Comparisons to singletons like None should always be done with is or is not, never the equality operators.